vowpalwabbit/core/src/parse_args.cc

```diff
@@ -1110,6 +1111,8 @@ void parse_example_tweaks(options_i& options, VW::workspace& all)
       .add(make_option("quantile_tau", quantile_loss_parameter)
                .default_value(0.5f)
                .help("Parameter \\tau associated with Quantile loss. Defaults to 0.5"))
+      .add(make_option("expectile_q", expectile_loss_parameter)
```

Comment:
Is  the default value `0`? If so, adding `.default_value(0.0f)` will be helpful to the end user

Reply:
It meant to be a "required" parameter when the expectile loss is used. We do not provide a default.
`0.0f` is actually invalid, and we check the range of the parameter value to make sure it is specified.

Reply:
Added "required" in help message.

Reply:
Do we check to make sure it is passed when quantile is being used?

Reply:
Sorry, for the required parameter for expectile loss do we error out if it isnt passed?

Reply:
Yes, we'll throw with an error message.